### ISSUE TITLE
Speed comparison for SVGs

### ISSUE THREAD

**Issue by samikeijonen** [WebDevStudios/wd_s#169]
SVGs is one of the best parts of your cool `wd_s`. I was wondering have you compared page speed between
-  Font Icons, let's say Genericons.
- Inline SVG sprite as you're using in `wd_s`.
- External SVG sprite with [svgxuse](https://github.com/Keyamoon/svgxuse).

I assume that inline SVG sprite is the fastest if you only have couple of icons. But if you have plenty of icons (30-50) I was wondering if external SVG sprite is faster because it can be cached.

Anyways love your work and I can try to compare speed between those three options myself.

**Comment by gregrickaby** [comment]
@samikeijonen We have not compared, but this sounds super interesting! I'd love to see the results.

We've had projects with 30+ icons, and yes, it would be great to cache that. I think though, if a user is running HTML caching then the icons sprite (which is written into the HTML) should be cache though?

I might need to put this through the paces and let you know. Thanks!

**Comment by samikeijonen** [comment]
> running HTML caching then the icons sprite (which is written into the HTML) should be cache though

This is what I have thought also but I'm definitely not a cache ninja.

I'll also try to create some kind of comparison.

**Comment by gregrickaby** [comment]
After spending an hour doing some research, here are my findings.
- It's not possible to "enqueue" `svg-icons.svg` The file itself must be written to the DOM.
- Prior to Chrome 37, all the svg "guts" needed to render after `<body>`
- That Chrome [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=349175) has been squashed! 
- Now, we can move the svg "guts" to the footer.

**Average results:**

47 icons embedded immediately after `<body>`
12 requests | 102KB transferred | Finish: 746ms | DomContentLoaded: 707ms | Load: 706ms

47 icons embedded immediately before `</body>`
12 requests | 102KB transferred | Finish: 598ms | DomContentLoaded: 570ms | Load: 570ms

**Bottom Line**
With `47` icons in the DOM, that's a `148ms` network load improvement and another `137ms` in render improvement!

I've gone ahead and moved the SVG definitions to the footer. .https://github.com/WebDevStudios/wd_s/commit/01336f8bb383f9c1540a3a15f14e91094be4f5b6

**Comment by samikeijonen** [comment]
Cool! Thanks for taking time for measuring it.
